cmd/explorer/main.go

```diff
@@ -377,102 +377,103 @@ func main() {
 			)
 
 			router.HandleFunc("/", handlers.Index).Methods("GET")
-			router.HandleFunc("/latestState", handlers.LatestState).Methods("GET")
-			router.HandleFunc("/launchMetrics", handlers.SlotVizMetrics).Methods("GET")
+			router.HandleFunc("/turnstile/verify", handlers.VerifyTurnstile).Methods("GET")
```

Comment:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/5877862/05987fb6-2ee5-4d88-b37f-6fa00ccf28b5)
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/5877862/5ff2699d-1f7a-40e8-b065-b2b4de933e97)

Modal appeared twice and I had to click "I am humand" twice. Hope this screenshots of my network traffic helps. I was on the index page.

Edit: Now the modal even appear three times